src/scripts/main.js

```diff
@@ -1,3 +1,249 @@
+const tbody = document.querySelector('tbody');
+const employeeList = tbody.querySelectorAll('tr');
+const thList = thead.querySelectorAll('th');
+const form = document.createElement('form');
+const formHtml = `
+  <label>Name:
+    <input name="name" type="text" data-qa="name">
+  </label>
+
+  <label>Position:
+    <input name="position" type="text" data-qa="position">
+  </label>
+
+  <label>Office:
+    <select name="office" data-qa="office">
+      <option value="Tokyo">Tokyo</option>
+      <option value="Singapore">Singapore</option>
+      <option value="London">London</option>
+      <option value="New York">New York</option>
+      <option value="Edinburgh">Edinburgh</option>
+      <option value="San Francisco">San Francisco</option>
+    </select>
+  </label>
+
+  <label>Age:
+    <input name="age" type="number" data-qa="age">
+  </label>
+
+  <label>Salary:
+    <input name="salary" type="number" data-qa="salary">
+  </label>
+
+  <button>Save to table</button>
+`;
+
+form.insertAdjacentHTML('afterbegin', formHtml);
+form.classList.add('new-employee-form');
+table.after(form);
+
+const submit = form.querySelector('button');
+
+function formatSalary(employee, index) {
+  const data = employee.cells[index].innerHTML;
+
+  if (data[0] === '$') {
+    return +data.split(',').join('').slice(1);
+  }
+
+  return data;
+}
+
+function formatVal(val) {
+  if (val) {
+    return val
+      .trim()
+      .split(' ')
+      .map(word => word[0].toUpperCase() + word.slice(1))
+      .join(' ');
+  }
+}
+
+function setOrderTrue(arr) {
+  arr.forEach(th => {
+    th.dataset.ascOrder = true;
+  });
+}
+
+function sortList(e) {
+  const columnIndex = e.target.cellIndex;
+  const currCol = e.target;
+  const falseList = thead.querySelectorAll('[data-asc-order="false"]');
+  let sortedArr;
+
+  if (currCol.dataset.ascOrder === 'true') {
+    setOrderTrue([...falseList]);
+
+    currCol.dataset.ascOrder = false;
+
+    sortedArr = [...employeeList].sort((employee1, employee2) => {
+      const data1 = formatSalary(employee1, columnIndex);
+      const data2 = formatSalary(employee2, columnIndex);
+
+      if (Number(data1)) {
+        return data1 - data2;
+      }
+
+      return data1.localeCompare(data2);
+    });
+  } else {
+    setOrderTrue([...falseList]);
+
+    sortedArr = [...employeeList].sort((employee1, employee2) => {
+      const data1 = formatSalary(employee1, columnIndex);
+      const data2 = formatSalary(employee2, columnIndex);
+
+      if (Number(data1)) {
+        return data2 - data1;
+      }
+
+      return data2.localeCompare(data1);
+    });
+  }
+
+  tbody.append(...sortedArr);
+}
+
+function selectRow(e) {
+  const isActive = tbody.querySelector('.active');
+  const targetRow = e.target.closest('tr');
+
+  if ([...targetRow.classList].includes('active')) {
+    isActive.classList.remove('active');
+
+    return;
+  }
+
+  if (isActive) {
+    isActive.classList.remove('active');
+  }
+
+  targetRow.classList.add('active');
```

Comment:
```suggestion
  if (![...targetRow.classList].includes('active')) {
    targetRow.classList.add('active');
  }

  if (isActive) {
    isActive.classList.remove('active');
  }
```

Reply:
@slavanosatskyi  oooohh...  I thought that we must have opportunity to deactivate current raw.